docs/case-worker-guide.md

```diff
@@ -51,24 +51,30 @@ This allows the user to upload a .csv file and issue many codes at once to a lis
 
 The bulk uploader is written as a javascript client. Rather than uploading the file, it parses on the client and makes requests to the issue API. This allows the server to discard phone numbers after sending the SMS without storing them. The uploader may be canceled and resumed, respects the server's rate-limiting throttle, and prevents the user from sending an SMS to the same phone twice using a retry code.
 
-### CSV Format
+#### CSV Format
```

Comment:
If we're gonna go 4 layers deep, can we add a TOC at the top? https://marketplace.visualstudio.com/items?itemName=AlanWalk.markdown-toc is good for automating if you use vscode.